manifests/init.pp

```diff
@@ -37,6 +42,14 @@
   $mesh_ipv6_prefixlen = ip_prefixlen($mesh_ipv6)
   $mesh_ipv6_address   = ip_address($mesh_ipv6)
 
+  # set default interface names
+  if $mesh_interface == "" {
+    $mesh_interface = "${mesh_code}-mesh-vpn"
```

Comment:
"-mesh-vpn" already has 9 characters, I think the limit for an interface name is 15 so only 6 chars left for each interface.

that's why I chose "-mvpn" instead